### ISSUE TITLE
Unnecessary heading markup is given to Self-hosted codespaces tree item. 

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode#106031]
["Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
**GitHubTags**:#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCode;#WCAG1.3.1; #A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;

Environment Details:​
Application Name: VS Codespace
Operating System: Win10​
Version: 1.47.0-insider (user setup)
Commit: d5e9aa0
Date: 2020-07-08T10:52:52.439Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References:WCAG1.3.1

**Repro Steps:​**

1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Codespace"
3. Click on "Visual Studio codespace".
4.Launch Narrator.
5.Navigate up to remote explorer section and click on it.
6. Navigate up to cloud-hosted codespaces  and self-hosted codespacestree item items and observed that narrator is announcing heading level or not.

**Actual:**
Unnecessary heading markup is given to cloud-hosted codespaces  and self-hosted codespaces tree item items

**Expected: ​**
Heading markup should be given to appropriate content. Heading markup should not be available for cloud-hosted codespaces  and self-hosted codespaces tree item items

**​User Impact: ​**
Screen reader user will not face difficulty to understand the actual content.

**Attachment link for Reference:**
[106031_Unnecessary heading markup is given to Self-hosted codespaces tree item..zip](https://github.com/microsoft/vscode/files/5168188/106031_Unnecessary.heading.markup.is.given.to.Self-hosted.codespaces.tree.item.zip)
